CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,248 @@
+
+#### Ignored assignments
+
+If a citizen misses _any_ assignment, their course grade is lowered by a half letter grade for each (e.g. _one_ (1) assignment missed would result in a `C-`, et al.).
+
+#### Academic dishonesty
+
+Citizens engaging in academically dishonest behavior will be assessed on a case-by-case basis, as circumstances and factors of a case vary widely. However, in situations where academic dishonesty is established, the variety of penalties might include (but are not limited to):
+
+* disqualification from the `A` and/or `B` "tiers"
+* an automatic assessment of an "Ignored" assignment
+
+In severe cases, academic dishonesty may result in the mark of an `F` for the course, though these cases are rare (i.e. _almost_ non-existent). For questions of what qualitfies under the provision of "academic dishonesty" see the remainder of this document and the _Compass_.
+
+### Assignment "bugs" and impact on grading
+
+* In the event that there are "bugs" with grader criteria, Mayor will assess completion status with known issues in mind
+  * The Mayor will make every effort to remedy the failing check before the assignment due date
+  * However, if during assignment evaluation, a check fails for every citizen due to a technical issue, the check is considered an invalid grading criteria   
+  
+### The "Rules Rule"
+
+* Convenants in this contract may be changed following:
+  * a successful "Pull Request" made on this document _and_
+  * an open course discussion about the proposed change(s)
+* Any changes will then be put to a vote
+  * If a vote succeeds with a majority (50% + 1) outcome supporting the measure(s) proposed, the change will be made 
+
+* The above applies to all portions of the contract _except_ for the "Basic Grade" and this rule ("Rules Rule")
+* The Mayor reserves the following rights:
+  * the right to veto a proposed change if:
+    * the change the academic or community integrity of the course, _or_ 
+    * a rule is superceded by policies governed by the College's academic standards or handbook
+  * the right make unilateral changes that that uphold academic and community integrity
+
+### Participation
+
+For the duration of this course, take "participation" to mean:
+
+* helping in group activities
+* completing assignments
+* Listens to class instruction
+* Giving full effort to class activities
+* Actively contributing to collaborative assignments
+* Being prepared for class by completing work outside of class hours
+* For group assignments, each member of the group should appear in the "Contributors" list
+* Respect neighborhood members' working hours
+  * If a member prefers not to work past 9:00p, recognize that their participation and time may be limited to a certain window each day and that this window may not always be the same
```

Comment:
We should find a method to post our schedule and work window for each person and update actively.